kura/org.eclipse.kura.network.threat.manager/OSGI-INF/flooding_configuration_service.xml

```diff
@@ -23,5 +23,5 @@
       <provide interface="org.eclipse.kura.security.FloodingProtectionConfigurationService"/>
    </service>
    <property name="kura.ui.service.hide" type="Boolean" value="true"/>
-   <reference bind="setFirewallConfigurationService" unbind="unsetFirewallConfigurationService" cardinality="1..1" interface="org.eclipse.kura.net.admin.FirewallConfigurationService" name="FirewallConfigurationService" policy="static"/> 
+   <reference bind="setFirewallConfigurationService" unbind="unsetFirewallConfigurationService" cardinality="1..1" interface="org.eclipse.kura.net.admin.FirewallConfigurationService" name="FirewallConfigurationService" policy="static"/>
```

Comment:
is there any non formatting-related change to this file?

Reply:
No, probably the auto-formatter has removed the trailing space.

Reply:
Then can you please either revert the changes to this file or update the copyright header?